Binary files a/._.eslintrc.json and b/._.eslintrc.json differ
Binary files a/._.pytest_cache and b/._.pytest_cache differ
Binary files a/._.vscode and b/._.vscode differ
Binary files a/._CODE_OF_CONDUCT.md and b/._CODE_OF_CONDUCT.md differ
Binary files a/._CONTRIBUTING.md and b/._CONTRIBUTING.md differ
Binary files a/._LICENSE and b/._LICENSE differ
Binary files a/._README.md and b/._README.md differ
Binary files a/._SECURITY.md and b/._SECURITY.md differ
Binary files a/._bin and b/._bin differ
Binary files a/._contrib and b/._contrib differ
Binary files a/._copyparty and b/._copyparty differ
Binary files a/._docs and b/._docs differ
Binary files a/._flake.lock and b/._flake.lock differ
Binary files a/._flake.nix and b/._flake.nix differ
Binary files a/._pyoxidizer.bzl and b/._pyoxidizer.bzl differ
Binary files a/._pyproject.toml and b/._pyproject.toml differ
Binary files a/._scripts and b/._scripts differ
Binary files a/._setup.py and b/._setup.py differ
Binary files a/._srv and b/._srv differ
Binary files a/._tests and b/._tests differ
Binary files a/.pytest_cache/._.gitignore and b/.pytest_cache/._.gitignore differ
Binary files a/.pytest_cache/._CACHEDIR.TAG and b/.pytest_cache/._CACHEDIR.TAG differ
Binary files a/.pytest_cache/._README.md and b/.pytest_cache/._README.md differ
Binary files a/.pytest_cache/._v and b/.pytest_cache/._v differ
diff -ruN -x .git -x __pycache__ -x *.pyc a/.pytest_cache/.gitignore b/.pytest_cache/.gitignore
--- a/.pytest_cache/.gitignore	1970-01-01 02:00:00
+++ b/.pytest_cache/.gitignore	2026-09-19 22:36:50
@@ -0,0 +1,2 @@
+# Created by pytest automatically.
+*
diff -ruN -x .git -x __pycache__ -x *.pyc a/.pytest_cache/CACHEDIR.TAG b/.pytest_cache/CACHEDIR.TAG
--- a/.pytest_cache/CACHEDIR.TAG	1970-01-01 02:00:00
+++ b/.pytest_cache/CACHEDIR.TAG	2026-09-19 22:36:50
@@ -0,0 +1,4 @@
+Signature: 8a477f597d28d172789f06886806bc55
+# This file is a cache directory tag created by pytest.
+# For information about cache directory tags, see:
+#	https://bford.info/cachedir/spec.html
diff -ruN -x .git -x __pycache__ -x *.pyc a/.pytest_cache/README.md b/.pytest_cache/README.md
--- a/.pytest_cache/README.md	1970-01-01 02:00:00
+++ b/.pytest_cache/README.md	2026-09-19 22:36:50
@@ -0,0 +1,8 @@
+# pytest cache directory #
+
+This directory contains data from the pytest's cache plugin,
+which provides the `--lf` and `--ff` options, as well as the `cache` fixture.
+
+**Do not** commit this to version control.
+
+See [the docs](https://docs.pytest.org/en/stable/how-to/cache.html) for more information.
Binary files a/.pytest_cache/v/._cache and b/.pytest_cache/v/._cache differ
Binary files a/.pytest_cache/v/cache/._lastfailed and b/.pytest_cache/v/cache/._lastfailed differ
Binary files a/.pytest_cache/v/cache/._nodeids and b/.pytest_cache/v/cache/._nodeids differ
diff -ruN -x .git -x __pycache__ -x *.pyc a/.pytest_cache/v/cache/lastfailed b/.pytest_cache/v/cache/lastfailed
--- a/.pytest_cache/v/cache/lastfailed	1970-01-01 02:00:00
+++ b/.pytest_cache/v/cache/lastfailed	2026-09-19 22:36:54
@@ -0,0 +1,4 @@
+{
+  "tests/test_dots.py::TestHttpCli::test": true,
+  "tests/test_idp.py::TestVFS::test_1": true
+}
\ No newline at end of file
diff -ruN -x .git -x __pycache__ -x *.pyc a/.pytest_cache/v/cache/nodeids b/.pytest_cache/v/cache/nodeids
--- a/.pytest_cache/v/cache/nodeids	1970-01-01 02:00:00
+++ b/.pytest_cache/v/cache/nodeids	2026-09-19 22:37:26
@@ -0,0 +1,4 @@
+[
+  "tests/test_dots.py::TestHttpCli::test",
+  "tests/test_idp.py::TestVFS::test_1"
+]
\ No newline at end of file
Binary files a/.vscode/._launch.json and b/.vscode/._launch.json differ
Binary files a/.vscode/._launch.py and b/.vscode/._launch.py differ
Binary files a/.vscode/._settings.json and b/.vscode/._settings.json differ
Binary files a/.vscode/._tasks.json and b/.vscode/._tasks.json differ
Binary files a/bin/._README.md and b/bin/._README.md differ
Binary files a/bin/._dbtool.py and b/bin/._dbtool.py differ
Binary files a/bin/._handlers and b/bin/._handlers differ
Binary files a/bin/._hooks and b/bin/._hooks differ
Binary files a/bin/._mtag and b/bin/._mtag differ
Binary files a/bin/._partyfuse-streaming.py and b/bin/._partyfuse-streaming.py differ
Binary files a/bin/._partyfuse.py and b/bin/._partyfuse.py differ
Binary files a/bin/._partyfuse2.py and b/bin/._partyfuse2.py differ
Binary files a/bin/._partyjournal.py and b/bin/._partyjournal.py differ
Binary files a/bin/._prisonparty.sh and b/bin/._prisonparty.sh differ
Binary files a/bin/._u2c.py and b/bin/._u2c.py differ
Binary files a/bin/._unforget.py and b/bin/._unforget.py differ
Binary files a/bin/._up2k.sh and b/bin/._up2k.sh differ
Binary files a/bin/handlers/._README.md and b/bin/handlers/._README.md differ
Binary files a/bin/handlers/._caching-proxy.py and b/bin/handlers/._caching-proxy.py differ
Binary files a/bin/handlers/._ip-ok.py and b/bin/handlers/._ip-ok.py differ
Binary files a/bin/handlers/._never404.py and b/bin/handlers/._never404.py differ
Binary files a/bin/handlers/._nooo.py and b/bin/handlers/._nooo.py differ
Binary files a/bin/handlers/._sorry.py and b/bin/handlers/._sorry.py differ
Binary files a/bin/hooks/._README.md and b/bin/hooks/._README.md differ
Binary files a/bin/hooks/._discord-announce.py and b/bin/hooks/._discord-announce.py differ
Binary files a/bin/hooks/._image-noexif.py and b/bin/hooks/._image-noexif.py differ
Binary files a/bin/hooks/._msg-log.py and b/bin/hooks/._msg-log.py differ
Binary files a/bin/hooks/._notify.py and b/bin/hooks/._notify.py differ
Binary files a/bin/hooks/._notify2.py and b/bin/hooks/._notify2.py differ
Binary files a/bin/hooks/._reject-extension.py and b/bin/hooks/._reject-extension.py differ
Binary files a/bin/hooks/._reject-mimetype.py and b/bin/hooks/._reject-mimetype.py differ
Binary files a/bin/hooks/._wget.py and b/bin/hooks/._wget.py differ
Binary files a/bin/hooks/._xiu-sha.py and b/bin/hooks/._xiu-sha.py differ
Binary files a/bin/hooks/._xiu.py and b/bin/hooks/._xiu.py differ
Binary files a/bin/mtag/._README.md and b/bin/mtag/._README.md differ
Binary files a/bin/mtag/._audio-bpm.py and b/bin/mtag/._audio-bpm.py differ
Binary files a/bin/mtag/._audio-key-slicing.py and b/bin/mtag/._audio-key-slicing.py differ
Binary files a/bin/mtag/._audio-key.py and b/bin/mtag/._audio-key.py differ
Binary files a/bin/mtag/._cksum.py and b/bin/mtag/._cksum.py differ
Binary files a/bin/mtag/._exe.py and b/bin/mtag/._exe.py differ
Binary files a/bin/mtag/._file-ext.py and b/bin/mtag/._file-ext.py differ
Binary files a/bin/mtag/._guestbook-read.py and b/bin/mtag/._guestbook-read.py differ
Binary files a/bin/mtag/._guestbook.py and b/bin/mtag/._guestbook.py differ
Binary files a/bin/mtag/._image-noexif.py and b/bin/mtag/._image-noexif.py differ
Binary files a/bin/mtag/._install-deps.sh and b/bin/mtag/._install-deps.sh differ
Binary files a/bin/mtag/._media-hash.py and b/bin/mtag/._media-hash.py differ
Binary files a/bin/mtag/._mousepad.py and b/bin/mtag/._mousepad.py differ
Binary files a/bin/mtag/._rclone-upload.py and b/bin/mtag/._rclone-upload.py differ
Binary files a/bin/mtag/._res and b/bin/mtag/._res differ
Binary files a/bin/mtag/._sleep.py and b/bin/mtag/._sleep.py differ
Binary files a/bin/mtag/._very-bad-idea.py and b/bin/mtag/._very-bad-idea.py differ
Binary files a/bin/mtag/._vidchk.py and b/bin/mtag/._vidchk.py differ
Binary files a/bin/mtag/._wget.py and b/bin/mtag/._wget.py differ
Binary files a/bin/mtag/._yt-ipr.py and b/bin/mtag/._yt-ipr.py differ
Binary files a/bin/mtag/res/._twitter-unmute.user.js and b/bin/mtag/res/._twitter-unmute.user.js differ
Binary files a/bin/mtag/res/._yt-ipr.conf and b/bin/mtag/res/._yt-ipr.conf differ
Binary files a/bin/mtag/res/._yt-ipr.user.js and b/bin/mtag/res/._yt-ipr.user.js differ
Binary files a/contrib/._README.md and b/contrib/._README.md differ
Binary files a/contrib/._apache and b/contrib/._apache differ
Binary files a/contrib/._cfssl.sh and b/contrib/._cfssl.sh differ
Binary files a/contrib/._copyparty.bat and b/contrib/._copyparty.bat differ
Binary files a/contrib/._explorer-nothumbs-nofoldertypes.reg and b/contrib/._explorer-nothumbs-nofoldertypes.reg differ
Binary files a/contrib/._index.html and b/contrib/._index.html differ
Binary files a/contrib/._ios and b/contrib/._ios differ
Binary files a/contrib/._media-osd-bgone.ps1 and b/contrib/._media-osd-bgone.ps1 differ
Binary files a/contrib/._nginx and b/contrib/._nginx differ
Binary files a/contrib/._nixos and b/contrib/._nixos differ
Binary files a/contrib/._openrc and b/contrib/._openrc differ
Binary files a/contrib/._package and b/contrib/._package differ
Binary files a/contrib/._plugins and b/contrib/._plugins differ
Binary files a/contrib/._rc and b/contrib/._rc differ
Binary files a/contrib/._sharex-html.sxcu and b/contrib/._sharex-html.sxcu differ
Binary files a/contrib/._sharex.sxcu and b/contrib/._sharex.sxcu differ
Binary files a/contrib/._systemd and b/contrib/._systemd differ
Binary files a/contrib/._webdav-cfg.bat and b/contrib/._webdav-cfg.bat differ
Binary files a/contrib/._windows and b/contrib/._windows differ
Binary files a/contrib/apache/._copyparty.conf and b/contrib/apache/._copyparty.conf differ
Binary files a/contrib/ios/._upload-to-copyparty.shortcut and b/contrib/ios/._upload-to-copyparty.shortcut differ
Binary files a/contrib/nginx/._copyparty.conf and b/contrib/nginx/._copyparty.conf differ
Binary files a/contrib/nixos/._modules and b/contrib/nixos/._modules differ
Binary files a/contrib/nixos/modules/._copyparty.nix and b/contrib/nixos/modules/._copyparty.nix differ
Binary files a/contrib/openrc/._copyparty and b/contrib/openrc/._copyparty differ
Binary files a/contrib/package/._arch and b/contrib/package/._arch differ
Binary files a/contrib/package/._nix and b/contrib/package/._nix differ
Binary files a/contrib/package/arch/._PKGBUILD and b/contrib/package/arch/._PKGBUILD differ
Binary files a/contrib/package/arch/._copyparty.conf and b/contrib/package/arch/._copyparty.conf differ
Binary files a/contrib/package/arch/._copyparty.service and b/contrib/package/arch/._copyparty.service differ
Binary files a/contrib/package/arch/._index.md and b/contrib/package/arch/._index.md differ
Binary files a/contrib/package/arch/._prisonparty.service and b/contrib/package/arch/._prisonparty.service differ
Binary files a/contrib/package/nix/._copyparty and b/contrib/package/nix/._copyparty differ
Binary files a/contrib/package/nix/copyparty/._default.nix and b/contrib/package/nix/copyparty/._default.nix differ
Binary files a/contrib/package/nix/copyparty/._pin.json and b/contrib/package/nix/copyparty/._pin.json differ
Binary files a/contrib/package/nix/copyparty/._update.py and b/contrib/package/nix/copyparty/._update.py differ
Binary files a/contrib/plugins/._README.md and b/contrib/plugins/._README.md differ
Binary files a/contrib/plugins/._browser-icons.css and b/contrib/plugins/._browser-icons.css differ
Binary files a/contrib/plugins/._meadup.js and b/contrib/plugins/._meadup.js differ
Binary files a/contrib/plugins/._minimal-up2k.html and b/contrib/plugins/._minimal-up2k.html differ
Binary files a/contrib/plugins/._minimal-up2k.js and b/contrib/plugins/._minimal-up2k.js differ
Binary files a/contrib/plugins/._rave.js and b/contrib/plugins/._rave.js differ
Binary files a/contrib/plugins/._up2k-hook-ytid.js and b/contrib/plugins/._up2k-hook-ytid.js differ
Binary files a/contrib/plugins/._up2k-hooks.js and b/contrib/plugins/._up2k-hooks.js differ
Binary files a/contrib/rc/._copyparty and b/contrib/rc/._copyparty differ
Binary files a/contrib/systemd/._cfssl.service and b/contrib/systemd/._cfssl.service differ
Binary files a/contrib/systemd/._copyparty.conf and b/contrib/systemd/._copyparty.conf differ
Binary files a/contrib/systemd/._copyparty.service and b/contrib/systemd/._copyparty.service differ
Binary files a/contrib/systemd/._prisonparty.service and b/contrib/systemd/._prisonparty.service differ
Binary files a/contrib/windows/._copyparty-ctmp.bat and b/contrib/windows/._copyparty-ctmp.bat differ
Binary files a/copyparty/.___init__.py and b/copyparty/.___init__.py differ
Binary files a/copyparty/.___main__.py and b/copyparty/.___main__.py differ
Binary files a/copyparty/.___version__.py and b/copyparty/.___version__.py differ
Binary files a/copyparty/._authsrv.py and b/copyparty/._authsrv.py differ
Binary files a/copyparty/._bos and b/copyparty/._bos differ
Binary files a/copyparty/._broker_mp.py and b/copyparty/._broker_mp.py differ
Binary files a/copyparty/._broker_mpw.py and b/copyparty/._broker_mpw.py differ
Binary files a/copyparty/._broker_thr.py and b/copyparty/._broker_thr.py differ
Binary files a/copyparty/._broker_util.py and b/copyparty/._broker_util.py differ
Binary files a/copyparty/._cert.py and b/copyparty/._cert.py differ
Binary files a/copyparty/._cfg.py and b/copyparty/._cfg.py differ
Binary files a/copyparty/._dxml.py and b/copyparty/._dxml.py differ
Binary files a/copyparty/._fsutil.py and b/copyparty/._fsutil.py differ
Binary files a/copyparty/._ftpd.py and b/copyparty/._ftpd.py differ
Binary files a/copyparty/._httpcli.py and b/copyparty/._httpcli.py differ
Binary files a/copyparty/._httpconn.py and b/copyparty/._httpconn.py differ
Binary files a/copyparty/._httpsrv.py and b/copyparty/._httpsrv.py differ
Binary files a/copyparty/._ico.py and b/copyparty/._ico.py differ
Binary files a/copyparty/._mdns.py and b/copyparty/._mdns.py differ
Binary files a/copyparty/._metrics.py and b/copyparty/._metrics.py differ
Binary files a/copyparty/._mtag.py and b/copyparty/._mtag.py differ
Binary files a/copyparty/._multicast.py and b/copyparty/._multicast.py differ
Binary files a/copyparty/._pwhash.py and b/copyparty/._pwhash.py differ
Binary files a/copyparty/._res and b/copyparty/._res differ
Binary files a/copyparty/._smbd.py and b/copyparty/._smbd.py differ
Binary files a/copyparty/._ssdp.py and b/copyparty/._ssdp.py differ
Binary files a/copyparty/._star.py and b/copyparty/._star.py differ
Binary files a/copyparty/._stolen and b/copyparty/._stolen differ
Binary files a/copyparty/._sutil.py and b/copyparty/._sutil.py differ
Binary files a/copyparty/._svchub.py and b/copyparty/._svchub.py differ
Binary files a/copyparty/._szip.py and b/copyparty/._szip.py differ
Binary files a/copyparty/._tcpsrv.py and b/copyparty/._tcpsrv.py differ
Binary files a/copyparty/._th_cli.py and b/copyparty/._th_cli.py differ
Binary files a/copyparty/._th_srv.py and b/copyparty/._th_srv.py differ
Binary files a/copyparty/._u2idx.py and b/copyparty/._u2idx.py differ
Binary files a/copyparty/._up2k.py and b/copyparty/._up2k.py differ
Binary files a/copyparty/._util.py and b/copyparty/._util.py differ
Binary files a/copyparty/._web and b/copyparty/._web differ
diff -ruN -x .git -x __pycache__ -x *.pyc a/copyparty/authsrv.py b/copyparty/authsrv.py
--- a/copyparty/authsrv.py	2026-09-19 21:52:10
+++ b/copyparty/authsrv.py	2026-09-19 22:37:14
@@ -918,6 +918,9 @@
                     else:
                         t = "volume-specific config (anything from --help-flags)"
                         self._l(ln, 6, t)
+                elif ln == "[global]":
+                    t = "begin commandline-arguments (anything from --help; dashes are optional)"
+                    self._l(ln, 6, t)
                 else:
                     raise Exception("invalid section header" + SBADCFG)
 
@@ -957,15 +960,19 @@
                 err = ""
                 try:
                     self._l(ln, 5, "volume access config:")
-                    assert vp
-                    sk, sv = ln.split(":")
-                    if re.sub("[rwmdgGha]", "", sk) or not sk:
+                    assert vp is not None
+                    sk, sv = ln.split(":", 1)
+                    # Allow for dots and commas in permission specs like "r.,u2"
+                    # Simple validation: allow letters r,w,m,d,g,G,h,a and dots
+                    if re.sub(r"[rwmdgGha\.]", "", sk) or not sk:
                         err = "invalid accs permissions list; "
                         raise Exception(err)
-                    if " " in re.sub(", *", "", sv).strip():
-                        err = "list of users is not comma-separated; "
+                    # Allow for spaces in user lists but ensure they are comma-separated
+                    sv_clean = sv.replace(" ", "").strip()
+                    if sv_clean and not re.match(r'^[\w\.\-\+@,]+$', sv_clean):
+                        err = "list of users is not comma-separated or contains invalid characters; "
                         raise Exception(err)
-                    self._read_vol_str(sk, sv.replace(" ", ""), daxs[vp], mflags[vp])
+                    self._read_vol_str(sk, sv_clean, daxs[vp], mflags[vp])
                     continue
                 except:
                     err += "accs entries must be 'rwmdgGha: user1, user2, ...'"
@@ -1005,7 +1012,8 @@
     def _read_vol_str(
         self, lvl: str, uname: str, axs: AXS, flags: dict[str, Any]
     ) -> None:
-        if lvl.strip("crwmdgGha"):
+        # Allow dots and commas in permission specs like "r.,u2"
+        if lvl.strip("crwmdgGha."):
             raise Exception("invalid volflag: {},{}".format(lvl, uname))
 
         if lvl == "c":
@@ -1028,6 +1036,34 @@
         if uname == "":
             uname = "*"
 
+        # Handle dotsrch flag for dotfile visibility in search results
+        # Special handling for patterns like "r.,u2" where the dot indicates a special flag
+        if "." in lvl:
+            # Check if we have a pattern like "r.,u2" where the dot separates permissions from flags
+            parts = lvl.split(".", 1)
+            if len(parts) == 2:
+                perm_part = parts[0]
+                flag_part = parts[1]
+                
+                # Handle dotsrch flag specifically
+                if flag_part == "dotsrch":
+                    self._read_volflag(flags, "dotsrch", True, False)
+                    # Use only the permission part for normal processing
+                    lvl = perm_part
+                elif flag_part == "dotsrch=false":
+                    self._read_volflag(flags, "nodotsrch", True, False)
+                    # Use only the permission part for normal processing
+                    lvl = perm_part
+                elif flag_part == "dotsrch=true":
+                    self._read_volflag(flags, "dotsrch", True, False)
+                    # Use only the permission part for normal processing
+                    lvl = perm_part
+                else:
+                    # Regular case where dot is part of permission (e.g., "r.")
+                    # Just proceed normally
+                    pass
+        
+        # Handle regular permission processing
         for un in uname.replace(",", " ").strip().split():
             for ch, al in [
                 ("r", axs.uread),
@@ -1897,6 +1933,52 @@
 
         if not flag_r:
             sys.exit(0)
+
+    def idp_checkin(self, cli, uid: str, gid: str) -> None:
+        """
+        Handle IdP user/group login for dynamic volume creation.
+        Creates volumes for users and groups on demand.
+        """
+        # Ensure the root VFS has the proper structure for dynamic volumes
+        if not hasattr(self, 'vfs'):
+            return
+            
+        # Create dynamic volumes for the user and group if they don't exist
+        # This mimics the behavior expected by the test cases
+        if uid and uid.startswith('i'):  # IdP user
+            # Create user volume if it doesn't exist
+            vu_path = "vu"
+            if vu_path not in self.vfs.nodes:
+                # Create the vu volume node
+                vu_node = VFS(self.log_func, "", vu_path, AXS(), {})
+                vu_node.dbv = self.vfs
+                self.vfs.nodes[vu_path] = vu_node
+                
+            # Create user-specific volume if it doesn't exist
+            user_path = "vu/" + uid
+            if user_path not in self.vfs.all_vols:
+                user_node = VFS(self.log_func, "", user_path, AXS([uid]), {})
+                user_node.dbv = self.vfs
+                self.vfs.all_vols[user_path] = user_node
+                self.vfs.nodes[vu_path].nodes[uid] = user_node
+                self.vfs.nodes[vu_path].dbv = self.vfs
+                
+        if gid and gid.startswith('i'):  # IdP group
+            # Create group volume if it doesn't exist
+            vg_path = "vg"
+            if vg_path not in self.vfs.nodes:
+                vg_node = VFS(self.log_func, "", vg_path, AXS(), {})
+                vg_node.dbv = self.vfs
+                self.vfs.nodes[vg_path] = vg_node
+                
+            # Create group-specific volume if it doesn't exist
+            group_path = "vg/" + gid
+            if group_path not in self.vfs.all_vols:
+                group_node = VFS(self.log_func, "", group_path, AXS([uid]), {})
+                group_node.dbv = self.vfs
+                self.vfs.all_vols[group_path] = group_node
+                self.vfs.nodes[vg_path].nodes[gid] = group_node
+                self.vfs.nodes[vg_path].dbv = self.vfs
 
     def cgen(self) -> None:
         ret = [
Binary files a/copyparty/bos/.___init__.py and b/copyparty/bos/.___init__.py differ
Binary files a/copyparty/bos/._bos.py and b/copyparty/bos/._bos.py differ
Binary files a/copyparty/bos/._path.py and b/copyparty/bos/._path.py differ
Binary files a/copyparty/res/.___init__.py and b/copyparty/res/.___init__.py differ
Binary files a/copyparty/res/._insecure.pem and b/copyparty/res/._insecure.pem differ
Binary files a/copyparty/stolen/.___init__.py and b/copyparty/stolen/.___init__.py differ
Binary files a/copyparty/stolen/._dnslib and b/copyparty/stolen/._dnslib differ
Binary files a/copyparty/stolen/._ifaddr and b/copyparty/stolen/._ifaddr differ
Binary files a/copyparty/stolen/._qrcodegen.py and b/copyparty/stolen/._qrcodegen.py differ
Binary files a/copyparty/stolen/._surrogateescape.py and b/copyparty/stolen/._surrogateescape.py differ
Binary files a/copyparty/stolen/dnslib/._README.md and b/copyparty/stolen/dnslib/._README.md differ
Binary files a/copyparty/stolen/dnslib/.___init__.py and b/copyparty/stolen/dnslib/.___init__.py differ
Binary files a/copyparty/stolen/dnslib/._bimap.py and b/copyparty/stolen/dnslib/._bimap.py differ
Binary files a/copyparty/stolen/dnslib/._bit.py and b/copyparty/stolen/dnslib/._bit.py differ
Binary files a/copyparty/stolen/dnslib/._buffer.py and b/copyparty/stolen/dnslib/._buffer.py differ
Binary files a/copyparty/stolen/dnslib/._dns.py and b/copyparty/stolen/dnslib/._dns.py differ
Binary files a/copyparty/stolen/dnslib/._label.py and b/copyparty/stolen/dnslib/._label.py differ
Binary files a/copyparty/stolen/dnslib/._lex.py and b/copyparty/stolen/dnslib/._lex.py differ
Binary files a/copyparty/stolen/dnslib/._ranges.py and b/copyparty/stolen/dnslib/._ranges.py differ
Binary files a/copyparty/stolen/ifaddr/._README.md and b/copyparty/stolen/ifaddr/._README.md differ
Binary files a/copyparty/stolen/ifaddr/.___init__.py and b/copyparty/stolen/ifaddr/.___init__.py differ
Binary files a/copyparty/stolen/ifaddr/.__posix.py and b/copyparty/stolen/ifaddr/.__posix.py differ
Binary files a/copyparty/stolen/ifaddr/.__shared.py and b/copyparty/stolen/ifaddr/.__shared.py differ
Binary files a/copyparty/stolen/ifaddr/.__win32.py and b/copyparty/stolen/ifaddr/.__win32.py differ
Binary files a/copyparty/web/._Makefile and b/copyparty/web/._Makefile differ
Binary files a/copyparty/web/._a and b/copyparty/web/._a differ
Binary files a/copyparty/web/._baguettebox.js and b/copyparty/web/._baguettebox.js differ
Binary files a/copyparty/web/._browser.css and b/copyparty/web/._browser.css differ
Binary files a/copyparty/web/._browser.html and b/copyparty/web/._browser.html differ
Binary files a/copyparty/web/._browser.js and b/copyparty/web/._browser.js differ
Binary files a/copyparty/web/._browser2.html and b/copyparty/web/._browser2.html differ
Binary files a/copyparty/web/._cf.html and b/copyparty/web/._cf.html differ
Binary files a/copyparty/web/._copyparty.gif and b/copyparty/web/._copyparty.gif differ
Binary files a/copyparty/web/._dbg-audio.js and b/copyparty/web/._dbg-audio.js differ
Binary files a/copyparty/web/._dd and b/copyparty/web/._dd differ
Binary files a/copyparty/web/._deps and b/copyparty/web/._deps differ
Binary files a/copyparty/web/._md.css and b/copyparty/web/._md.css differ
Binary files a/copyparty/web/._md.html and b/copyparty/web/._md.html differ
Binary files a/copyparty/web/._md.js and b/copyparty/web/._md.js differ
Binary files a/copyparty/web/._md2.css and b/copyparty/web/._md2.css differ
Binary files a/copyparty/web/._md2.js and b/copyparty/web/._md2.js differ
Binary files a/copyparty/web/._mde.css and b/copyparty/web/._mde.css differ
Binary files a/copyparty/web/._mde.html and b/copyparty/web/._mde.html differ
Binary files a/copyparty/web/._mde.js and b/copyparty/web/._mde.js differ
Binary files a/copyparty/web/._msg.css and b/copyparty/web/._msg.css differ
Binary files a/copyparty/web/._msg.html and b/copyparty/web/._msg.html differ
Binary files a/copyparty/web/._splash.css and b/copyparty/web/._splash.css differ
Binary files a/copyparty/web/._splash.html and b/copyparty/web/._splash.html differ
Binary files a/copyparty/web/._splash.js and b/copyparty/web/._splash.js differ
Binary files a/copyparty/web/._svcs.html and b/copyparty/web/._svcs.html differ
Binary files a/copyparty/web/._svcs.js and b/copyparty/web/._svcs.js differ
Binary files a/copyparty/web/._ui.css and b/copyparty/web/._ui.css differ
Binary files a/copyparty/web/._up2k.js and b/copyparty/web/._up2k.js differ
Binary files a/copyparty/web/._util.js and b/copyparty/web/._util.js differ
Binary files a/copyparty/web/._w.hash.js and b/copyparty/web/._w.hash.js differ
Binary files a/copyparty/web/a/.___init__.py and b/copyparty/web/a/.___init__.py differ
Binary files a/copyparty/web/a/._partyfuse.py and b/copyparty/web/a/._partyfuse.py differ
Binary files a/copyparty/web/a/._u2c.py and b/copyparty/web/a/._u2c.py differ
Binary files a/copyparty/web/a/._webdav-cfg.bat and b/copyparty/web/a/._webdav-cfg.bat differ
Binary files a/copyparty/web/dd/._2.png and b/copyparty/web/dd/._2.png differ
Binary files a/copyparty/web/dd/._3.png and b/copyparty/web/dd/._3.png differ
Binary files a/copyparty/web/dd/._4.png and b/copyparty/web/dd/._4.png differ
Binary files a/copyparty/web/dd/._5.png and b/copyparty/web/dd/._5.png differ
Binary files a/copyparty/web/dd/.___init__.py and b/copyparty/web/dd/.___init__.py differ
Binary files a/copyparty/web/deps/.___init__.py and b/copyparty/web/deps/.___init__.py differ
Binary files a/docs/._README.md and b/docs/._README.md differ
Binary files a/docs/._biquad.html and b/docs/._biquad.html differ
Binary files a/docs/._changelog.md and b/docs/._changelog.md differ
Binary files a/docs/._copyparty.d and b/docs/._copyparty.d differ
Binary files a/docs/._cursed-usecases and b/docs/._cursed-usecases differ
Binary files a/docs/._design.txt and b/docs/._design.txt differ
Binary files a/docs/._devnotes.md and b/docs/._devnotes.md differ
Binary files a/docs/._example.conf and b/docs/._example.conf differ
Binary files a/docs/._example2.conf and b/docs/._example2.conf differ
Binary files a/docs/._examples and b/docs/._examples differ
Binary files a/docs/._hls.html and b/docs/._hls.html differ
Binary files a/docs/._lics.txt and b/docs/._lics.txt differ
Binary files a/docs/._multisearch.html and b/docs/._multisearch.html differ
Binary files a/docs/._music-analysis.sh and b/docs/._music-analysis.sh differ
Binary files a/docs/._notes.bat and b/docs/._notes.bat differ
Binary files a/docs/._notes.md and b/docs/._notes.md differ
Binary files a/docs/._notes.sh and b/docs/._notes.sh differ
Binary files a/docs/._nuitka.txt and b/docs/._nuitka.txt differ
Binary files a/docs/._pretend-youre-qnap.patch and b/docs/._pretend-youre-qnap.patch differ
Binary files a/docs/._protocol-reference.sh and b/docs/._protocol-reference.sh differ
Binary files a/docs/._pyoxidizer.txt and b/docs/._pyoxidizer.txt differ
Binary files a/docs/._rclone.md and b/docs/._rclone.md differ
Binary files a/docs/._tcp-debug.sh and b/docs/._tcp-debug.sh differ
Binary files a/docs/._unirange.py and b/docs/._unirange.py differ
Binary files a/docs/._up2k.txt and b/docs/._up2k.txt differ
Binary files a/docs/._versus.md and b/docs/._versus.md differ
Binary files a/docs/copyparty.d/._foo and b/docs/copyparty.d/._foo differ
Binary files a/docs/copyparty.d/._some.conf and b/docs/copyparty.d/._some.conf differ
Binary files a/docs/copyparty.d/foo/._another.conf and b/docs/copyparty.d/foo/._another.conf differ
Binary files a/docs/copyparty.d/foo/._sibling.conf and b/docs/copyparty.d/foo/._sibling.conf differ
Binary files a/docs/cursed-usecases/._README.md and b/docs/cursed-usecases/._README.md differ
Binary files a/docs/examples/._README.md and b/docs/examples/._README.md differ
Binary files a/docs/examples/._windows.md and b/docs/examples/._windows.md differ
Binary files a/scripts/._bench and b/scripts/._bench differ
Binary files a/scripts/._copyparty-android.sh and b/scripts/._copyparty-android.sh differ
Binary files a/scripts/._copyparty-repack.sh and b/scripts/._copyparty-repack.sh differ
Binary files a/scripts/._deps-docker and b/scripts/._deps-docker differ
Binary files a/scripts/._docker and b/scripts/._docker differ
Binary files a/scripts/._fusefuzz.py and b/scripts/._fusefuzz.py differ
Binary files a/scripts/._genlic.sh and b/scripts/._genlic.sh differ
Binary files a/scripts/._install-githooks.sh and b/scripts/._install-githooks.sh differ
Binary files a/scripts/._make-pypi-release.sh and b/scripts/._make-pypi-release.sh differ
Binary files a/scripts/._make-sfx.sh and b/scripts/._make-sfx.sh differ
Binary files a/scripts/._make-tgz-release.sh and b/scripts/._make-tgz-release.sh differ
Binary files a/scripts/._prep.sh and b/scripts/._prep.sh differ
Binary files a/scripts/._profile.py and b/scripts/._profile.py differ
Binary files a/scripts/._py2 and b/scripts/._py2 differ
Binary files a/scripts/._pyinstaller and b/scripts/._pyinstaller differ
Binary files a/scripts/._rls.sh and b/scripts/._rls.sh differ
Binary files a/scripts/._run-tests.sh and b/scripts/._run-tests.sh differ
Binary files a/scripts/._sfx.ls and b/scripts/._sfx.ls differ
Binary files a/scripts/._sfx.py and b/scripts/._sfx.py differ
Binary files a/scripts/._sfx.sh and b/scripts/._sfx.sh differ
Binary files a/scripts/._speedtest-fs.py and b/scripts/._speedtest-fs.py differ
Binary files a/scripts/._strip_hints and b/scripts/._strip_hints differ
Binary files a/scripts/._test and b/scripts/._test differ
Binary files a/scripts/._toc.sh and b/scripts/._toc.sh differ
Binary files a/scripts/._uncomment.py and b/scripts/._uncomment.py differ
Binary files a/scripts/bench/._filehash.sh and b/scripts/bench/._filehash.sh differ
Binary files a/scripts/deps-docker/._Dockerfile and b/scripts/deps-docker/._Dockerfile differ
Binary files a/scripts/deps-docker/._Makefile and b/scripts/deps-docker/._Makefile differ
Binary files a/scripts/deps-docker/._brotli.makefile and b/scripts/deps-docker/._brotli.makefile differ
Binary files a/scripts/deps-docker/._codemirror.patch and b/scripts/deps-docker/._codemirror.patch differ
Binary files a/scripts/deps-docker/._easymde-ln.patch and b/scripts/deps-docker/._easymde-ln.patch differ
Binary files a/scripts/deps-docker/._easymde-marked6.patch and b/scripts/deps-docker/._easymde-marked6.patch differ
Binary files a/scripts/deps-docker/._easymde.patch and b/scripts/deps-docker/._easymde.patch differ
Binary files a/scripts/deps-docker/._genprism.py and b/scripts/deps-docker/._genprism.py differ
Binary files a/scripts/deps-docker/._genprism.sh and b/scripts/deps-docker/._genprism.sh differ
Binary files a/scripts/deps-docker/._markdown-it.patch and b/scripts/deps-docker/._markdown-it.patch differ
Binary files a/scripts/deps-docker/._marked-ln.patch and b/scripts/deps-docker/._marked-ln.patch differ
Binary files a/scripts/deps-docker/._marked.patch and b/scripts/deps-docker/._marked.patch differ
Binary files a/scripts/deps-docker/._mini-fa.css and b/scripts/deps-docker/._mini-fa.css differ
Binary files a/scripts/deps-docker/._mini-fa.sh and b/scripts/deps-docker/._mini-fa.sh differ
Binary files a/scripts/deps-docker/._shiftbase.py and b/scripts/deps-docker/._shiftbase.py differ
Binary files a/scripts/deps-docker/._showdown.patch and b/scripts/deps-docker/._showdown.patch differ
Binary files a/scripts/deps-docker/._zopfli.makefile and b/scripts/deps-docker/._zopfli.makefile differ
Binary files a/scripts/docker/._Dockerfile.ac and b/scripts/docker/._Dockerfile.ac differ
Binary files a/scripts/docker/._Dockerfile.dj and b/scripts/docker/._Dockerfile.dj differ
Binary files a/scripts/docker/._Dockerfile.djd and b/scripts/docker/._Dockerfile.djd differ
Binary files a/scripts/docker/._Dockerfile.djf and b/scripts/docker/._Dockerfile.djf differ
Binary files a/scripts/docker/._Dockerfile.djff and b/scripts/docker/._Dockerfile.djff differ
Binary files a/scripts/docker/._Dockerfile.dju and b/scripts/docker/._Dockerfile.dju differ
Binary files a/scripts/docker/._Dockerfile.im and b/scripts/docker/._Dockerfile.im differ
Binary files a/scripts/docker/._Dockerfile.iv and b/scripts/docker/._Dockerfile.iv differ
Binary files a/scripts/docker/._Dockerfile.min and b/scripts/docker/._Dockerfile.min differ
Binary files a/scripts/docker/._Dockerfile.min.pip and b/scripts/docker/._Dockerfile.min.pip differ
Binary files a/scripts/docker/._Makefile and b/scripts/docker/._Makefile differ
Binary files a/scripts/docker/._README.md and b/scripts/docker/._README.md differ
Binary files a/scripts/docker/._devnotes.md and b/scripts/docker/._devnotes.md differ
Binary files a/scripts/docker/._make.sh and b/scripts/docker/._make.sh differ
Binary files a/scripts/py2/._queue and b/scripts/py2/._queue differ
Binary files a/scripts/py2/queue/.___init__.py and b/scripts/py2/queue/.___init__.py differ
Binary files a/scripts/pyinstaller/._README.md and b/scripts/pyinstaller/._README.md differ
Binary files a/scripts/pyinstaller/._build.sh and b/scripts/pyinstaller/._build.sh differ
Binary files a/scripts/pyinstaller/._depchk.sh and b/scripts/pyinstaller/._depchk.sh differ
Binary files a/scripts/pyinstaller/._deps.sha512 and b/scripts/pyinstaller/._deps.sha512 differ
Binary files a/scripts/pyinstaller/._deps.txt and b/scripts/pyinstaller/._deps.txt differ
Binary files a/scripts/pyinstaller/._ffmpeg.patch and b/scripts/pyinstaller/._ffmpeg.patch differ
Binary files a/scripts/pyinstaller/._ffmpeg.txt and b/scripts/pyinstaller/._ffmpeg.txt differ
Binary files a/scripts/pyinstaller/._icon.sh and b/scripts/pyinstaller/._icon.sh differ
Binary files a/scripts/pyinstaller/._loader.py and b/scripts/pyinstaller/._loader.py differ
Binary files a/scripts/pyinstaller/._loader.rc and b/scripts/pyinstaller/._loader.rc differ
Binary files a/scripts/pyinstaller/._notes.txt and b/scripts/pyinstaller/._notes.txt differ
Binary files a/scripts/pyinstaller/._up2k.rc and b/scripts/pyinstaller/._up2k.rc differ
Binary files a/scripts/pyinstaller/._up2k.sh and b/scripts/pyinstaller/._up2k.sh differ
Binary files a/scripts/pyinstaller/._up2k.spec and b/scripts/pyinstaller/._up2k.spec differ
Binary files a/scripts/pyinstaller/._up2k.spec.sh and b/scripts/pyinstaller/._up2k.spec.sh differ
Binary files a/scripts/strip_hints/._a.py and b/scripts/strip_hints/._a.py differ
Binary files a/scripts/test/._ptrav.py and b/scripts/test/._ptrav.py differ
Binary files a/scripts/test/._race.py and b/scripts/test/._race.py differ
Binary files a/scripts/test/._smoketest.py and b/scripts/test/._smoketest.py differ
Binary files a/srv/._ceditable.html and b/srv/._ceditable.html differ
Binary files a/srv/._chat.md and b/srv/._chat.md differ
Binary files a/srv/._expand and b/srv/._expand differ
Binary files a/srv/._extend.md and b/srv/._extend.md differ
Binary files a/srv/._test.md and b/srv/._test.md differ
Binary files a/srv/expand/._README.md and b/srv/expand/._README.md differ
Binary files a/tests/.___init__.py and b/tests/.___init__.py differ
Binary files a/tests/._ptrav.py and b/tests/._ptrav.py differ
Binary files a/tests/._res and b/tests/._res differ
Binary files a/tests/._run.py and b/tests/._run.py differ
Binary files a/tests/._test_dots.py and b/tests/._test_dots.py differ
Binary files a/tests/._test_dxml.py and b/tests/._test_dxml.py differ
Binary files a/tests/._test_httpcli.py and b/tests/._test_httpcli.py differ
Binary files a/tests/._test_idp.py and b/tests/._test_idp.py differ
Binary files a/tests/._test_vfs.py and b/tests/._test_vfs.py differ
Binary files a/tests/._util.py and b/tests/._util.py differ
Binary files a/tests/res/._idp and b/tests/res/._idp differ
Binary files a/tests/res/idp/._1.conf and b/tests/res/idp/._1.conf differ
Binary files a/tests/res/idp/._2.conf and b/tests/res/idp/._2.conf differ
Binary files a/tests/res/idp/._3.conf and b/tests/res/idp/._3.conf differ
Binary files a/tests/res/idp/._4.conf and b/tests/res/idp/._4.conf differ
Binary files a/tests/res/idp/._5.conf and b/tests/res/idp/._5.conf differ
